client/state/signup/steps/jpo-connect/action.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+const debug = require( 'debug' )( 'calypso:jetpack-jpo:actions' );
+
+/**
+ * Internal dependencies
+ */
+import { SIGNUP_STEPS_JPO_CONNECT_SET } from 'state/action-types';
+
+export function setJPOConnect( connect ) {
```

Comment:
It's hard for me to tell what the `connect` param represents here. It might help to have some jsdoc for the function, or a more explanatory name for the parameter.

Reply:
I agree. Perhaps @rcoll can write up a JS doc here, as I'm still learning about the flow.